api/lib/src/lib.rs

```diff
@@ -362,7 +362,8 @@ pub trait BrokerApi: SignedExtrinsicApi {
 /// chain.
 pub fn clean_foreign_chain_address(chain: ForeignChain, address: &str) -> Result<EncodedAddress> {
 	Ok(match chain {
-		ForeignChain::Ethereum => EncodedAddress::Eth(clean_hex_address(address)?),
+		ForeignChain::Ethereum | ForeignChain::Arbitrum =>
+			EncodedAddress::Eth(clean_hex_address(address)?),
```

Comment:
Should this not be Evm? or should we add an extra variant Arb? We use EncodedAddress in some events, and not having the distinction I think will be a problem i.e. 


```
		LiquidityDepositAddressReady {
			channel_id: ChannelId,
			asset: Asset,
			deposit_address: EncodedAddress,
			// account the funds will be credited to upon deposit
			account_id: T::AccountId,
			deposit_chain_expiry_block: <AnyChain as Chain>::ChainBlockNumber,
			boost_fee: BasisPoints,
		},
```

Reply:
yeah, this is done by mistake, will change.